src/layouts/PublicLayout/PublicLayout.tsx

```diff
@@ -0,0 +1,132 @@
+/* Copyright (c) Fortanix, Inc.
+|* This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of
+|* the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+import { classNames as cx, type ComponentProps } from '../../util/componentUtil.ts';
+import { useScroller } from '../util/Scroller.tsx';
+
+import { type IconName } from '../../components/graphics/Icon/Icon.tsx';
+//import { FortanixLogo } from '../../fortanix/FortanixLogo/FortanixLogo.tsx';
+import { Card } from '../../components/containers/Card/Card.tsx';
+
+import fortanixLogo from '../../assets/fortanix/fortanix-logo.svg';
+import cl from './PublicLayout.module.scss';
+
+
+export { cl as PublicLayoutClassNames };
+
+type FortanixArmorLogoProps = ComponentProps<'figure'> & {
+  /** Whether to stack the logo and product name. */
+  stacked?: undefined | boolean,
+};
+const FortanixArmorLogo = (props: FortanixArmorLogoProps) => {
+  const { stacked = false, ...propsRest } = props;
+  return (
+    <figure
+      {...propsRest}
+      className={cx(
+        cl['bk-fortanix-armor-logo'],
+        { [cl['bk-fortanix-armor-logo--stacked']]: stacked },
+        propsRest.className,
+    )}
+    >
+      {/* <FortanixLogo/> */}
```

Comment:
Any specific reason not to use the `<FortanixLogo/>`? I assume it's due to the more horizontal nature of the image, with Armor written on the side, instead of the bottom. Eventually that could be a variation of said component. Not a deal breaker, of course.

Reply:
It's blocked by this issue: https://github.com/fortanix/baklava/issues/213

Basically we can't really scale the component at the moment, I had been using `zoom` as a workaround but turns out that doesn't work well in Safari. So for now I'm just using the SVG.

Reply:
Oh I see, thanks :)